tensorflow_addons/seq2seq/tests/attention_wrapper_test.py

```diff
@@ -297,683 +266,626 @@ def setUp(self):
+    cell = tf.keras.layers.LSTMCell(dummy_data.units)
+    attention_layer = tf.keras.layers.Dense(
+        dummy_data.units * 2, use_bias=False, activation=tf.math.tanh
+    )
+    attention_wrapper = wrapper.AttentionWrapper(
+        cell, attention_mechanism, attention_layer=attention_layer
+    )
+    with pytest.raises(ValueError):
+        # Should fail because the attention mechanism has not been
+        # initialized.
+        attention_wrapper.get_initial_state(
+            batch_size=dummy_data.batch, dtype=tf.float32
+        )
+    attention_mechanism.setup_memory(
+        dummy_data.encoder_outputs.astype(np.float32),
+        memory_sequence_length=dummy_data.encoder_sequence_length,
+    )
+    initial_state = attention_wrapper.get_initial_state(
+        batch_size=dummy_data.batch, dtype=tf.float32
+    )
+    assert initial_state.attention.shape[-1] == dummy_data.units * 2
+    first_input = dummy_data.decoder_inputs[:, 0].astype(np.float32)
+    output, _ = attention_wrapper(first_input, initial_state)
+    assert output.shape[-1] == dummy_data.units * 2
+
+
+def _test_with_attention(
+    create_attention_mechanism,
+    expected_final_output,
+    expected_final_state,
+    attention_mechanism_depth=3,
+    alignment_history=False,
+    expected_final_alignment_history=None,
+    attention_layer_size=6,
+    attention_layer=None,
+    create_query_layer=False,
+    create_memory_layer=True,
+    create_attention_kwargs=None,
+):
+    attention_layer_sizes = (
+        [attention_layer_size] if attention_layer_size is not None else None
+    )
+    attention_layers = [attention_layer] if attention_layer is not None else None
+    create_attention_mechanisms = [create_attention_mechanism]
+    attention_mechanism_depths = [attention_mechanism_depth]
+    assert len(create_attention_mechanisms) == 1
+    encoder_sequence_length = [3, 2, 3, 1, 1]
+    decoder_sequence_length = [2, 0, 1, 2, 3]
+    batch_size = 5
+    encoder_max_time = 8
+    decoder_max_time = 4
+    input_depth = 7
+    encoder_output_depth = 10
+    cell_depth = 9
+    create_attention_kwargs = create_attention_kwargs or {}
+
+    if attention_layer_sizes is not None:
+        # Compute sum of attention_layer_sizes. Use encoder_output_depth if
+        # None.
+        attention_depth = sum(
+            attention_layer_size or encoder_output_depth
+            for attention_layer_size in attention_layer_sizes
+        )
+    elif attention_layers is not None:
+        # Compute sum of attention_layers output depth.
+        attention_depth = sum(
+            attention_layer.compute_output_shape(
+                [batch_size, cell_depth + encoder_output_depth]
+            )
+            .dims[-1]
+            .value
+            for attention_layer in attention_layers
         )
+    else:
+        attention_depth = encoder_output_depth * len(create_attention_mechanisms)
 
-    def _testWithMaybeMultiAttention(
-        self,
-        is_multi,
```

Comment:
So we remove all logic for handling `is_multi`. It was never used in the test suite, but I'm less familiar with this module and don't know if it'll ever be needed in testing going forward. Okay to remove IMO since it wasn't used and will be in git history.